style.css

```diff
@@ -550,3 +552,105 @@
   border-radius: 8px;
   background-color: #2230d2;
 }
+
+.contact-me {
+  background-image: url(/img/Contact-bg2.svg);
+  background-position: top right;
+  background-repeat: no-repeat;
+  height: 779px;
+  width: 375px;
+  background-color: #6070ff;
+  border-top-left-radius: 100px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-left: 24px;
+  padding-right: 24px;
+  padding-top: 103px;
+}
```

Comment:
## Styling

- [ ]  think it would be better to adjust your background-image URL, it is not visible on your web page, 👍 Kindly try to follow the[ Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=UD1kG41aCI5N6m7W-0) as much as possible 🙏 
- [ ] It would be better also to change the ```color``` of your input text to match the Figma design as much as possible,
   -  Kindly the placeholder and the normal input text have different font color

- [ ] I think it would be better also to follow the Figma design's button as much as possible as ```Get in touch``` is not on the same line, Kindly try to fix this issue
    - It would be better also to include the Button interactions such as hover, active (when it's pressed), disabled, and enable (default state) styled as shown in the Figma template. 👍 👍 
    You may follow this [link](https://www.freecodecamp.org/news/css-button-style-hover-color-and-background/) for more info



 **Your design** | **Figma design**
 | --- | --- |
![image](https://user-images.githubusercontent.com/102757126/212969653-edee5b3f-0ef1-4ecb-ada5-5e3d7dd8d9a5.png) | ![image](https://user-images.githubusercontent.com/102757126/212969798-13527f2b-6b20-429e-af0f-f31c49e4e4e2.png)


## Figma Button design
![image](https://user-images.githubusercontent.com/102757126/212970031-761e10eb-a110-4c8b-89c9-d0c0fd8326fc.png)


### KEEP IT UP 👍 👍

Reply:
Fixed.